zerver/lib/rate_limiter.py

```diff
@@ -138,8 +139,9 @@ def rules(self) -> list[tuple[int, int]]:
 
 
 class RateLimitedIPAddr(RateLimitedObject):
-    def __init__(self, ip_addr: str, domain: str = "api_by_ip") -> None:
+    def __init__(self, ip_addr: str, domain: str, ipv6_prefix: int = 64) -> None:
```

Comment:
Is there any reasons why we're deleting `api_by_ip`? I think it's fairly important to leave it there as the default value.

Reply:
The `domain` parameter represents which time window and No.requests should be used, defined in `DEFAULT_RATE_LIMITING_RULES`.

This must be different for other IPv6 prefixes like `/56`, so I think it's better to explicitly tell which domain is used any time we block an IP.

However and since this PR imposes a single prefix only `/64`, It's maybe better to leave the default value as it is.